### PR TITLE
feat(pipeline): in-cluster pipeline job create k8sclient directly

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
in-cluster pipeline job create k8sclient directly

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  in-cluster pipeline job create k8sclient directly            |
| 🇨🇳 中文    |    当前集群的流水线任务直连k8s          |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by chengjoey** [comment]
image

**Review by iutx — APPROVED** [review]
/lgtm

File: modules/pipeline/conf/conf.go [hunk 1]
```diff
@@ -24,9 +24,9 @@ import (
 
 // Conf 定义配置对象.
 type Conf struct {
-	ListenAddr  string `env:"LISTEN_ADDR" default:":3081"`
-	Debug       bool   `env:"DEBUG" default:"false"`
-	DiceCluster string `env:"DICE_CLUSTER" default:"local"` // 服务所在集群
```
Comment (sfwn) [inline modules/pipeline/conf/conf.go:29]: what's different between `DICE_CLUSTER` and `DICE_CLUSTER_NAME`?

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/pipeline/conf/conf.go (hunk 1 above)
Comment (chengjoey) [reply modules/pipeline/conf/conf.go:29]: not found `DICE_CLUSTER` in envs

**Review by sfwn — APPROVED** [review]

**Pull request merged**